### DISCUSSION TITLE
请教如何自己整账号

### DISCUSSION THREAD

**Discussion by POP-LOL123** [MagicPigZHN/Exhentai-Shared-Account#1] [Q&A] [answered]
大佬在 Readme 里面有提到自费注册了账号，想请教一下是如何自费注册的呢？任何可用的教程都会被感激！ @MagicPigZHN

**Comment by Mpigest** [comment, marked as answer]
> 大佬在 Readme 里面有提到自费注册了账号，想请教一下是如何自费注册的呢？任何可用的教程都会被感激！ @MagicPigZHN

- 在我的<a href=https://sleazyfork.org/zh-CN/scripts/465444-exhentai%E5%85%AC%E5%85%B1%E8%B4%A6%E5%8F%B7/discussions/185786>Sleazyfork</a>有给出对相似提问的解答；<p>
- 至于你提到的“自费”主要是指作者花钱购买了一个谷歌邮箱用于注册e站，因为据作者所知国内的邮箱大概率不能够获取里站权限；<p>
- 其次，作者使用机场的节点经测试可以获取里站权限（机场名不方便透露）。有能力也可以自建节点，但要注意节点所在区不能是亚洲，否则也不能获取里站权限；<p>
- 最后，想要详细的操作步骤作者推荐参考以下两篇教程：
1. <a href=https://nicebowl.fun/11>[E绅士里站访问教程——从注册账号到进入里站](https://nicebowl.fun/11)</a>
2. <a href=https://exhentai.home.blog/2021/10/24/jc01/>智障也能看懂的E站里站账号注册教程</a>

**Reply by POP-LOL123** [reply]
感谢大佬，那个其实也是我问的hh，但是当时网页上不去我以为插件页面被下架了~

**Reply by Mpigest** [reply]
have fun : )

**Reply by shjsjsn** [reply]
谢谢你，我登上了exhentai，我爱你，爹、

**Reply by Mpigest** [reply]
you deserved，have fun : )

**Comment by yunqiwanglin** [comment]
无语了，试了好几次，最后还是找某宝《根刻默离》弄了个
